test/ng/directive/inputSpec.js

```diff
@@ -657,17 +657,20 @@ describe('input', function() {
       expect($rootScope.form.alias.$error.month).toBeTruthy();
     });
 
-    it('should allow four or more digits in year', function() {
-      var inputElm = helper.compileInput('<input type="month" ng-model="value"  ng-model-options="{timezone: \'UTC\'}"/>');
 
-      helper.changeInputValueTo('10123-03');
```

Comment:
@AmazingJaze I assumed that Edge simply doesn't allow setting years with more than 4 digits in all the date input elements, so when you enter `10123-03` etc. the input has no value (because it's invalid) and we don't get a value to work with. But I haven't actually checked if this is an Edge bug.